_docs/12-pcb-assembly.md

```diff
@@ -111,13 +110,11 @@ title: PCB Assembly
 
 ![Arduino header pins fully soldered](/assets/images/Arduino_header_soldered.jpg)
 
-Carefully remove PCB from arduino
+![Carefully remove PCB from arduino]()
```

Comment:
Is this supposed to be an image or a text comment?

Reply:
I don't know; I just made it consistent with the others.